src/Mod/PartDesign/Gui/TaskSketchBasedParameters.h

```diff
@@ -61,6 +61,8 @@ class TaskSketchBasedParameters : public PartDesignGui::TaskFeatureParameters,
     QVariant objectNameByLabel(const QString& label, const QVariant& suggest) const;
 
     QString getFaceReference(const QString& obj, const QString& sub) const;
+    QString make2DLabel(const App::DocumentObject* section,

```

Comment:
Please add a note what the function does. This helps in the IDE a lot since the IDEs will take this info and display it as tooltips